pkg/apis/extensions/v1beta1/types.go

```diff
@@ -910,6 +910,9 @@ type ReplicaSetStatus struct {
 	// The number of pods that have labels matching the labels of the pod template of the replicaset.
 	FullyLabeledReplicas int32 `json:"fullyLabeledReplicas,omitempty" protobuf:"varint,2,opt,name=fullyLabeledReplicas"`
 
+	// The number of available replicas for this replica set.
+	AvailableReplicas int32 `json:"availableReplicas,omitempty" protobuf:"varint,4,opt,name=availableReplicas"`
```

Comment:
We have two concepts for pods-- live, and ready, I would use one of those terms. What does "Available" mean?

What do you intend to use this field for?